includes/class-freemius.php

```diff
@@ -509,23 +547,69 @@ private function _register_account_hooks() {
 		 * @uses   fs_find_caller_plugin_file
 		 */
 		private function _find_caller_plugin_file() {
-			// Try to load the cached value of the file path.
+			return $this->_storage->plugin_main_file->path;
+		}
+
+
+		/**
+		 * Identifies the caller type: plugin or theme.
+		 *
+		 * @author Leo Fajardo (@leorw)
+		 * @since  1.2.0
+		 *
+		 * @return string
```

Comment:
Ok, I see now. I was confused by the diff. I only looked at the `_find_caller_plugin_file` method and didn't notice the `@return string` at the bottom of the diff :)